### PR TITLE
Add Chinese (Simplified) Translation

### PR DESCRIPTION
**Pull request by jimmypury**
The interface has been translated into Chinese (Simplified).

Some words cannot be translated such as `Search (Ctrl+F)` or `Run Command (F2)` in the pic.
image
image

And there is a weird issue. I used to name the language json file to `zh_CN.json`, but it is failed to take effect (it does not work! And neither does the `pt_BR.json`.). So I rename it to `zh.json`, it works well.

### CODE REVIEW

File: default/config.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
         "font-size": "16px"
     },
     "highlight-modes": "",
+    "lang": "",
```
Comment (brrd) [inline default/config.json:29]: Could you please explain why you added this line?

**Review by brrd — COMMENTED** [review]
> Some words cannot be translated such as Search (Ctrl+F) or Run Command (F2) in the pic.

Thanks for reporting. See #300.

> And there is a weird issue. I used to name the language json file to zh_CN.json, but it is failed to take effect (it does not work! And neither does the pt_BR.json.). So I rename it to zh.json, it works well.

Could you please open a new issue about this, explaining how to reproduce this issue? We also need to know your OS and its version.

We received another similar PR a few days after yours: #299. You submitted first so I will give you priority. However feel free to read it and potentially improve some of your translations if you think it's relevant.

I will wait for your answer to merge this PR.

File: default/config.json (hunk 1 above)
Comment (jimmypury) [reply default/config.json:29]: Is it redundant?
I think the configuration file should contain the necessary settings.
If it is an error, just ignore it please, cause I am a Freshman and it is hard for me to read JavaScript.

Comment (brrd) [reply default/config.json:29]: Well, I think this is out of the scope of this PR, but it sounds ok. I'll merge it.

**Pull request merged**